doc/protocol/set.md

```diff
@@ -0,0 +1,191 @@
+## set: Write data
+
+KUKSA.val supports basic set requests according to VISS  validated against the following schema.
+
+
+```json
+{
```

Comment:
better reference to https://github.com/eclipse/kuksa.val/blob/master/include/VSSRequestJsonSchema.hpp#L48-L71 instead of copy the code?